src/app/screens/Accounts/Show/index.tsx

```diff
@@ -0,0 +1,592 @@
+import {
```

Comment:
nit: not sure if `Show` is the right wording for this. What about `DetailView`?

Reply:
> What about `DetailView`?

Sounds good @bumi wdyt?

Reply:
+1 for detail. However I'd probably drop the `View` part and just have `src/app/screens/Accounts/Detail/index.tsx`

Reply:
I've already renamed the Publisher Show Screen (the only other similar screen) in this PR itself to align things, should I change it in this itself?

Reply:
No, I'd keep this PR as is and do the rename in a separate PR like @bumi suggested.